app/models/response.rb

```diff
@@ -7,6 +7,7 @@
 #  gif_url        :string
 #  not_working    :boolean          default(FALSE)
 #  notices        :jsonb
+#  productivity   :integer
```

Comment:
Shouldn't there be a validation for what is an acceptable range of values for `productivity`?

Reply:
Sorry, I thought I was supposed to review this now. Please only assign reviewers when you are ready for them to do that.

Reply:
@LyubovP I agree with Roger you should add validation for `productivity`.

Reply:
I added a validation for productivity, only if the steps array includes the string "ProductivityBadFollowUp". Since productivity is only present on the "ProductivityBadFollowUp" step.

Reply:
@LyubovP it would be nice if you added validation at the model level, that productivity has values ​​from 0 to 9 (do not forget about the rspec tests please). 
It may also be necessary to add the ability to edit productivity in AA, check this with the QA this please.